docs/source/configuration/internalproxy.md

```diff
@@ -1,28 +1,42 @@
-# Internal proxy to API
+# Internal proxy to content backend API
+
+The server side Volto SSR process (based on Razzle) has an internal proxy to the backend API
+enabled by default.
 
-While in development, Volto has an internal proxy to the backend API enabled by default.
 It provides a better developer experience out of the box, so the developer doesn't has to
 deal with [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) and can focus on
 develop/test drive/demo Volto.
 
+To understand the need for the internal proxy, there are three processes running in a Volto website:
+
+1. A frontend web application running in your browser (Javascript)
+2. A Node.js server process that delivers the javascript to the client and does
+   Server Side Generation (SSR) of your pages on first request (Javascript, the
+   Razzle package is used for SSR)
+3. A Plone server process that stores and delivers all content through a REST API (Python)
+
 ## Configuration
 
-By default, Volto expects a Plone backend located at `http://localhost:8080/Plone`.
-These are the settings that allows you to configure the API and how the proxy works:
+The default values from Volto configuration expect a Plone content backend located at `http://localhost:8080/Plone`. Below are the settings that allows you to change the its location
 
-- `apiPath` [API_PATH] - URL of the backend, used through Volto. By default, the proxy URL.
-- `devProxyToApiPath` - The real backend URL, used by the proxy. By default, `http://localhost:8080/Plone`
+- `apiPath` [API_PATH] - URL of the backend, used by Volto running in the frontend browser. By default, the proxy URL. (http://localhost:3000/++api++/)
```

Comment:
This changed overtime, `apiPath` in the config is no longer meant to be "touched" but controlled via environment variables. 

Also, the apiPath is no longer used in the proxy...

So I would rewrite the entire section using only `devProxyToApiPath` and/or RAZZLE_DEV_PROXY_API_PATH.